.github/pull_request_template.md

```diff
@@ -0,0 +1,38 @@
+## Changes
+
+_Explanation of changes goes here_
+
+## Notes
+
+_Any other notes go here_
+
+## To Do
+
+_Any remaining things that need to get done_
+
+- [ ] item 1
+- [ ] ...
+
+## Checklist
+
+It can be helpful to check the `Checks` and `Files changed` tabs.
+Please review the [contributor guide](https://github.com/Northeastern-Electric-Racing/PM-Dashboard-v2/blob/main/docs/ContributorGuide.md) and reach out to your squad if anything is unclear.
+
+- [ ] All commits are tagged with the ticket number
+- [ ] No linting errors
+- [ ] No newline at end of file warnings
+- [ ] All code follows repository-configured prettier formatting
+- [ ] No merge conflicts
+- [ ] All checks passing
+- [ ] Code sufficiently tested
+- [ ] Screenshots of UI changes (if applicable)
+- [ ] Remove any not-applicable sections
+- [ ] Assign the PR to yourself
+- [ ] PR is linked to the ticket
+- [ ] Request reviewers & ping on Slack
+
```

Comment:
I'd say add something about double checking that they haven't committed the package-lock.json, cause that's a pretty common error. Also maybe a note about making sure that you've been working with an up to date version of the code, but that one's less important. Lastly, I think moving the request reviewers and ping on slack part to line 19, and saying something like "please request reviewers and ping on slack only after you've gone through this whole checklist" would hammer home the problem you're trying to solve with this template.

Reply:
ooop yes